src/apps/run/components/juttle-viewer.js

```diff
@@ -15,10 +15,14 @@ class JuttleViewer extends Component {
     }
 
     render() {
+        if (!this.props.juttleSource) {
```

Comment:
prolly better to rename this to `bundle` for consistency, or if the viewer only needs the program, pass `bundle.program` into the prop